tests/InputFiles/Cce/CharacteristicExtract.yaml

```diff
@@ -24,73 +29,117 @@ ResourceInfo:
           SafetyFactor: 0.9
 
+  # The number of angular modes used by the CCE evolution. This must be larger
+  # than ObservationLMax. We always use all of the m modes for the LMax since
+  # using fewer m modes causes aliasing-driven instabilities.
   LMax: 20
-  NumberOfRadialPoints: 12
+  # Probably don't need more than 15 radial grid points, but could increase
+  # up to ~20
+  NumberOfRadialPoints: 15
+  # The maximum ell we use for writing waveform output. While CCE can dump
+  # more, you should be cautious with higher modes since mode mixing, truncation
+  # error, and systematic numerical effects can have significant contamination
+  # in these modes.
   ObservationLMax: 8
 
   InitializeJ:
+    # To see what other J-initialization procedures are available, comment
+    # out this group of options and do, e.g. "Blah:" The code will print
+    # an error message with the available options and a help string.
+    # More details can be found at spectre-code.org.
     ConformalFactor:
       AngularCoordTolerance: 1e-13
-      MaxIterations: 1000
-      RequireConvergence: False
+      MaxIterations: 1000 # Do extra iterations in case we improve.
+      RequireConvergence: False # Often don't converge to 1e-13, but that's fine
       OptimizeL0Mode: True
       UseBetaIntegralEstimate: False
       ConformalFactorIterationHeuristic: SpinWeight1CoordPerturbation
       UseInputModes: False
       InputModes: []
 
-  StartTime: 0.0
-  EndTime: 1000.0
+  StartTime: Auto # Start at the first time in file
+  EndTime: Auto   # End at the last time in file
   BoundaryDataFilename: CceR0257.h5
+  # WARNING: The ExtractionRadius must be updated with the file name!
+  ExtractionRadius: 257.0
+  # Assume data is metric & derivs. This is wrong for SpECTRE & SpEC
+  # output, but likely correct for other codes.
   H5IsBondiData: False
+  # How we interpolate the worldtube data in time for CCE.
   H5Interpolator:
     BarycentricRationalSpanInterpolator:
       MinOrder: 10
       MaxOrder: 10
-  ExtractionRadius: 257.0
+  # Unless you are using data from an old version of the Spectral Einstein Code,
+  # leave FixSpecNormalization as False.
   FixSpecNormalization: False
 
+  # Loads this many time steps in from the HDF5 files at once. Fewer file system
+  # accesses improve performance, but requires more RAM.
   H5LookaheadTimes: 10000
 
   Filtering:
-    RadialFilterHalfPower: 24
+    # Using half-power 64 means we effectively have a Heavidside filter, zeroing
+    # out the highest mode only.
+    RadialFilterHalfPower: 64
     RadialFilterAlpha: 35.0
+    # The number of angular modes not touched by the angular filter. This should
+    # be about 2 smaller than LMax used for the CCE evolution.
     FilterLMax: 18
 
   ScriInterpOrder: 5
-  ScriOutputDensity: 5
+  # How often per CCE time step to output. Given the tight default tolerances of
+  # 1e-6 with an Adams-Bashforth stepper, once per time step is probably fine.
+  ScriOutputDensity: 1
```

Comment:
@keefemitman Is this often enough for waveform analysis? I don't have an intuition on what we need

Reply:
`1` seems to be fine for the majority of runs I've looked at